src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -908,30 +933,30 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 			// Run action on Enter/Space
 			if (event.equals(KeyCode.Enter) || event.equals(KeyCode.Space)) {
 				handled = true;
-				const editor = this.group.getEditorByIndex(index);
+				const editor = this.tabsModel.getEditorByIndex(tabIndex);
 				if (editor) {
-					this.group.openEditor(editor);
+					this.groupViewer.openEditor(editor);
 				}
 			}
 
 			// Navigate in editors
-			else if ([KeyCode.LeftArrow, KeyCode.RightArrow, KeyCode.UpArrow, KeyCode.DownArrow, KeyCode.Home, KeyCode.End].some(kb => event.equals(kb))) {
-				let targetIndex: number;
+			else if ([KeyCode.LeftArrow, KeyCode.RightArrow, KeyCode.UpArrow, KeyCode.DownArrow, KeyCode.Home, KeyCode.End].some(kb => event.equals(kb))) { // TODO check
```

Comment:
I am fine leaving this up for a later PR: users can still tab to the other tab stripe and use arrows there to navigate.